controller/controller.go

```diff
@@ -39,6 +39,10 @@ const (
 	trueString  = "true"
 )
 
+var (
```

Comment:
This is a const.

Reply:
Added a detailed comment, I can see us changing this default at a per-binary level